src/components/ControlPanel.vue

```diff
@@ -1290,17 +1290,23 @@ export default {
 
         // wait a moment before refresh to check if the node
         // has been added to the group correctly
-        setTimeout(this.getAssociations, 500)
+        setTimeout(this.getAssociations, 1000)
```

Comment:
why a second, might this be more with bigger networks?

Reply:
@chrisns Yep, sometimes I have to reselect the group in order to get the associations, giving it some more time could prevent this :) Unfortunally there are no callbacks in zwave methods

Reply:
can we poll until it reaches the desired state or just throw messaging to the user to say the change has been submitted?

Reply:
@chrisns Sometimes it could happen that the association call isn't added/removed, so nope we could just retry it some times, or add a refresh button to manually refresh current associations

Reply:
sad face!